doc/rst/source/explain_-q_full.rst_

```diff
@@ -22,6 +22,7 @@ modifiers to control how the rows are counted [Default is **+a**]:
 Alternatively, use **+c**\ *col* to indicate that the arguments instead are min/max *data limits* for the values in
 column *col*. With **+c**\ *col*, only rows whose data for the given column *col* are within the range(s) given by the
 *min*/*max* limits are read (with **-qi**) or written (with **-qo**). **Note**: Because arguments may contain colons or
-be negative, your must specify *start*/*stop* instead of *start*\ [:*inc*]:*stop*.
+be negative, your must specify *start*/*stop* instead of *start*\ [:*inc*]:*stop*. If *min* or *max* is not given we
+default to -infinity and + infinity, respectively (e.g., **-qo**\ 50/**+c**\ 2 will only write records whose z-values is ≥ 50).
```

Comment:
```suggestion
default to -infinity and + infinity, respectively (e.g., **-qo**\ 50/**+c**\ 2 will only write records whose z-values (in 3rd column) is ≥ 50
while *-qo**\ /50**+c**\ 2 will only write records whose z-values is ≤ 50).
```
Is the inequality for the suggested example correct here (i.e., equal than is included in both directions) or should it be < 50?

Reply:
Yes, both ways.